File: app/src/main/java/org/jay/launchstarterexample/tasks/delayinittask/DelayInitTaskB.java
package org.jay.launchstarterexample.tasks.delayinittask;


import org.jay.launchstarter.MainTask;
import org.jay.launchstarterexample.utils.LogUtils;

public class DelayInitTaskB extends MainTask {

    @Override
    public void run() {
        // 模拟一些操作

        try {
            Thread.sleep(200);
        } catch (InterruptedException e) {
            e.printStackTrace();
        }
        LogUtils.i("DelayInitTaskB finished");
    }
}

File: app/src/main/java/org/jay/appstarterexample/tasks/delayinittask/DelayInitTaskB.kt
package org.jay.appstarterexample.tasks.delayinittask

import org.jay.appstarterexample.utils.LogUtils.i
import org.jay.appstarter.MainTask

class DelayInitTaskB : MainTask() {

    override fun run() {
        // 模拟一些操作
        try {
            Thread.sleep(200)
        } catch (e: InterruptedException) {
            e.printStackTrace()
        }
        i("DelayInitTaskB finished")
    }
}